mkdocs.yml

```diff
@@ -98,6 +98,9 @@ nav:
           - Locations: 'integrations/azure/locations.md'
           - Discovery: 'integrations/azure/discovery.md'
           - Org Data: 'integrations/azure/org.md'
+      - Azure BlobStorage:
+          - Locations: 'integrations/azureBlobstorage/locations.md'
+          - Discovery: 'integrations/azureBlobstorage/discovery.md'
```

Comment:
Assuming you want these to show up in the documentation at https://backstage.io/docs/overview/what-is-backstage, you need to modify `microsite/sidebar.json` instead.

This `mkdocs.yml` file only adds the files to the sample techdocs at https://demo.backstage.io/docs/default/component/backstage